consumer/src/main/java/com/flipkart/varadhi/consumer/MessageBatch.java

```diff
@@ -0,0 +1,33 @@
+package com.flipkart.varadhi.consumer;
+
+import java.util.List;
+
+public class MessageBatch {
```

Comment:
since we get messages in batches from the consumer, its better to just append those batches rather then doing the extra copy to move everything to single batch.